kuma/core/utils.py

```diff
@@ -284,3 +286,9 @@ def chord_flow(pre_task, tasks, post_task):
         return chain(*tasks)
     else:
         return chain(pre_task, chord(header=tasks, body=post_task))
+
+
+def limit_banned_ip_to_0(group, request):
+    if IPBan.objects.active(ip=get_ip(request)).count() > 0:
```

Comment:
Please use `exists` instead of `count() > 0` since `exists` has a slightly improved internal cache